sky/backends/onprem_utils.py

```diff
@@ -131,6 +131,12 @@ def get_local_auth_config(cluster_name: str) -> List[str]:
     return config['auth']
 
 
+def get_admin_python(cluster_name: str) -> List[str]:
+    """Returns admin python's path."""
```

Comment:
Do away with the concept "admin python".  It's just the global python executable path located on all nodes, executable by all users.

Same elsewhere in this change.

Reply:
Done, thanks!